nodejs/launcher/server/server.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const {
+  lightstep,
+  opentelemetry,
+} = require('lightstep-opentelemetry-launcher-node');
+
+const PORT = process.env.PORT || 8080;
+
+const sdk = lightstep.configureOpenTelemetry();
+
+sdk.start().then(() => {
+  const express = require('express');
+  const app = express();
+  app.use(express.json());
+
+  app.get('/', (req, res) => {
+    res.send('running...');
+  });
+
+  app.get('/ping', (req, res) => {
+    console.log(req.rawHeaders);
```

Comment:
If you find this too chatty, it can be removed. It's a remnant from some previous debugging I was doing.